src/com/pentalog/pentastagiu/homework2/Client.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class Client {
+    String FirstName;
+    String LastName;
+    double debit = 0;
+
+    public void deposit_money( int sum)
+    {
+        debit=debit+sum;
+        System.out.println("Done! Your account is " + debit + " now.");
+    }
+
+    public void withdraw_money(int sum)
+    {
```

Comment:
dar daca si in cazul asta e introdusa o suma negativa? e ok sa creasca balanta din cont in urma unei actiuni de withdraw? pentru a fi consistenti as zice ca trebuie adaugata validare si aici